Sources/Private/Utility/Primitives/VectorsExtensions.swift

```diff
@@ -339,3 +339,26 @@ extension CATransform3D {
     CATransform3DConcat(CATransform3D.makeSkew(skew: skew, skewAxis: skewAxis), self)
   }
 }
+
+// MARK: - CATransform3D + Equatable
+
+extension CATransform3D: Equatable {
+  public static func ==(lhs: CATransform3D, rhs: CATransform3D) -> Bool {
+    lhs.m11 == rhs.m11
+      && lhs.m12 == rhs.m12
+      && lhs.m13 == rhs.m13
+      && lhs.m14 == rhs.m14
+      && lhs.m21 == rhs.m21
+      && lhs.m22 == rhs.m22
+      && lhs.m23 == rhs.m23
+      && lhs.m24 == rhs.m24
+      && lhs.m31 == rhs.m31
+      && lhs.m32 == rhs.m32
+      && lhs.m33 == rhs.m33
+      && lhs.m34 == rhs.m34
+      && lhs.m41 == rhs.m41
+      && lhs.m42 == rhs.m42
+      && lhs.m43 == rhs.m43
+      && lhs.m44 == rhs.m44
```

Comment:
Can we use `CATransform3DEqualToTransform` instead?

Reply:
Aha thanks! Not used to the naming schemes of these helpers so I didn't find that